src/common/components/help/help-custom.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component, PropTypes } from 'react';
+
+import { HeadingThree } from '../vanilla/heading/';
+import styles from './help.css';
+
+
+export default class HelpCustom extends Component {
```

Comment:
New component to allow custom content in help box (without install instructions).